opencti-platform/opencti-front/src/private/components/data/csvMapper/CsvMapperForm.tsx

```diff
@@ -157,15 +168,15 @@ const CsvMapperForm: FunctionComponent<CsvMapperFormProps> = ({
           );
 
           return (
-            <Form style={{ margin: '20px 0 20px 0' }}>
+            <Form className={classes.formContainer}>
               <Field
                 component={TextField}
                 variant="standard"
                 name="name"
                 label={t('Name')}
                 fullWidth
               />
-              <div className={classes.center} style={{ marginTop: 20 }}>
+              <div className={`${classes.center} ${classes.marginTop}`}>
```

Comment:
Because there is some element you need to center without the margin.
We can keep it like this or create another class to merge this into one.
What is your advice ?

Reply:
Like you, I'm fine with having a different class for different concerns.

Reply:
For class customization we use classnames, can you use it ?